### ISSUE TITLE
Monitoring API: Endpoint metric_data Status 500 in dev mode

### ISSUE THREAD

**Issue by t-book** [GeoNode/geonode#4986]
## Expected Behavior
Valid results for:
http://localhost:8000/monitoring/api/metric_data/response.error.count/?last=600&interval=600

## Actual Behavior
![Bildschirmfoto 2019-10-03 um 12 43 07](https://user-images.githubusercontent.com/20478652/66120605-8a283480-e5db-11e9-8ba9-af58903c42c0.png)

## Steps to Reproduce the Problem

  1. start geonode with paver
  2. visit endpoint or monitoring view

## Specifications

  - GeoNode version: master
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker): paver start (django runserver)
  - Platform: Ubuntu Desktop
  - Additional details: Not quite familiar with monitoring app, could be that I miss something with settings

**Comment by simboss** [comment]
@gioscarda can you look into this?

**Comment by gioscarda** [comment]
Hi @t-book, it is due to the SQLite db you are using. The SQLite db is currently not supported by the monitoring app, with postgres it works fine.

**Comment by gioscarda** [comment]
Yes, we should set `MONITORING_ENABLED` and `USER_ANALYTICS_ENABLED` to `False` by default.

We should update the [documentation page](http://docs.geonode.org/en/2.10.x/intermediate/monitoring/geonode_monitoring/index.html#installation) too.

**Comment by capooti** [comment]
@gioscarda agreed. Any reason why this is not working on sqlite? I'd love to see it working there too and enabled by default.
On a side note, is it possible to disable the corresponding geoserver jars when not enabled?

**Comment by t-book** [comment]
> Hi @t-book, it is due to the SQLite db you are using. The SQLite db is currently not supported by the monitoring app, with postgres it works fine.

thanks @gioscarda for me it's fine to know that it's just sqlite. Unsure (@capooti) but I think we could close here?

**Comment by gioscarda** [comment]
> @gioscarda agreed. Any reason why this is not working on sqlite? I'd love to see it working there too and enabled by default.

@capooti , when calling that endpoint, GeoNode performs a raw SQL query which syntax is not fully compliant with the SQLite one. The `_covert_sql_params` method is used to fix the syntax for `SQLIte`, but it only changes the parameters injection:
https://github.com/GeoNode/geonode/blob/5dac976b26702cb370d7e6e0cbae8a6dec5151e6/geonode/utils.py#L1286-L1291
I just did some tests and it seems the problem is due to conditions on datetime fields in the `where` clause:
- `valid_from >= TIMESTAMP :valid_from AT TIME ZONE 'UTC'` --> not working with SQLite (current syntax)
 - `valid_from >= datetime(:valid_from)` --> working with SQLite

So I will open a new issue to disable monitoring by default.

> On a side note, is it possible to disable the corresponding geoserver jars when not enabled?

From what I know geoserver jars can be managed on the geoserver side only, but I would let @afabiani answer on this.

**Comment by capooti** [comment]
@gioscarda thanks for the informative answer. I didn't notice that monitoring is using raw sql queries.
